### ISSUE TITLE
Matt Beaudry - Issue

### ISSUE THREAD

**Issue by Mattibdry** [Mattibdry/exploding-robot-head#1]
Here's my code

**Comment by Mattibdry** [comment]
@thomasjbradley

**Comment by thomasjbradley** [comment]
Hmm... try this version out and see if it helps:
https://dl.dropboxusercontent.com/u/2506703/Markbot/Markbot-5.0.3-prerelease.dmg

**Comment by Mattibdry** [comment]
Hey Thomas,

I get this : https://gyazo.com/dd35b7945b4567a23e4f294d8eeafe57  when I click on the link you sent.

Matt

**Comment by thomasjbradley** [comment]
Yeah I deleted it because it was working for other people. I've released the full version to everybody now at the regular download URL:

https://dl.dropboxusercontent.com/u/2506703/Markbot/Install%20Markbot.dmg

**Comment by Mattibdry** [comment]
Okay perfect! Yeah it works now. Anyways, thanks! 👍
